test/suggestion.cpp

```diff
@@ -329,8 +333,32 @@ namespace {
     // should be included in the result documents.
     std::vector<std::string> resultSet = getSuggestions(archive, "she and the apple", archive.getEntryCount());
     std::vector<std::string> expectedResult = {
-                                                "she and the apple",
+                                                "she and the apple"
                                               };
     ASSERT_EQ(expectedResult, resultSet);
   }
+
+  TEST(Suggestion, checkRedirectionCollapse) {
+    TempZimArchive tza("testZim");
+    zim::writer::Creator creator;
+    creator.configIndexing(true, "en");
+    creator.startZimCreation(tza.getPath());
+
+    auto item = std::make_shared<TestItem>("testPath", "text/html", "Article Target");
+    creator.addItem(item);
+    creator.addRedirection("redirectionPath1", "Article Redirect 1", "testPath");
+    creator.addRedirection("redirectionPath2", "Article Redirect 2", "testPath");
```

Comment:
Please add redirection chain and be explicit in the tests that we are NOT fixing the chain.

(And if later we fix the chaining, we will update this test to show that we ARE fixing the chain).

Reply:
Adding a separate test which checks the current expected behavior. I have added a comment there specifying the nature of the test.